src/MangaJet/app/src/main/java/com/mangajet/mangajet/MainActivity.kt

```diff
@@ -135,4 +153,49 @@ class MainActivity : AppCompatActivity(), ActivityResultCallback<Map<String, Boo
         setupActionBarWithNavController(navController, appBarConfiguration)
         navView.setupWithNavController(navController)
     }
+
+    // Functions that checks report for crash => sends it to email
+    // Functions that checks report for crash => sends it to email
+    private fun checkForCrash(report : String){
+
+        val preferences = PreferenceManager.getDefaultSharedPreferences(this)
+        if(report.isEmpty() || preferences.getBoolean("CrashNeverAskAgain", false))
```

Comment:
I think button "NeverAskAgain" shouldn`t exist.
Just if person said "No" we should change preferences and let him use app.